cluster/addons/fluentd-elasticsearch/es-image/elasticsearch_logging_discovery.go

```diff
@@ -104,7 +105,7 @@ func main() {
 	var endpoints *api.Endpoints
 	addrs := []string{}
 	// Wait for some endpoints.
-	count := 0
+	count, _ := strconv.Atoi(os.Getenv("MINIMUM_MASTER_NODES"))
```

Comment:
Where is this environment variable being set?   I would imagine it should be done in the elastic search pod spec but I don't see that change.

Reply:
Nevermind, I see it in `run.sh` now.